app/views/users/_profile_card.html.erb

```diff
@@ -5,7 +5,7 @@
         <div class="row">
           <div class="col-md-5 align-items-center text-center">
             <div class="user-avatar">
-              <%= image_tag gravatar_url(user, 80), alt: 'gravatar' %>
+              <%= image_tag 'odin-logo.svg', class: 'logo-img' %>
```

Comment:
can we replace `'odin-logo.svg'` with `@user.image` please? that will use the users avatar pulled in from github.